### PR TITLE
fix(store): support moving blocks in multiple frames

### PR DESCRIPTION
**Pull request by fundon**
Try [heavy-whiteboard](https://blocksuite-toeverything.vercel.app/?init=heavy-whiteboard) example in page mode.

### CODE REVIEW

**Review by lawvs — DISMISSED** [review]
Could you add some comments to help us understand it better?

File: packages/store/src/workspace/page.ts [hunk 1]
```diff
@@ -389,44 +389,65 @@ export class Page extends Space<FlatBlockMap> {
...
-      }
+      // after the target
+      let offset = 1;
 
-      const ids = blocks.map(block => block.id);
-      if (insertBeforeSibling) {
-        yChildrenB.insert(nextIdx, ids);
-      } else {
-        yChildrenB.insert(nextIdx + 1, ids);
+      for (const [parent, blocks] of tempMap) {
```
Comment (doodlewind) [inline packages/store/src/workspace/page.ts:422]: It seems that each array value of the `tempMap` here only has one item (line 411), when do we need to loop through the `blocks` here?

**Comment by fundon** [comment]
> Could you add some comments to help us understand it better?
Added

File: packages/store/src/workspace/page.ts (hunk 1 above)
Comment (fundon) [reply packages/store/src/workspace/page.ts:486]: This is the case in the `heavy-whiteboard` example, but we can edit the frames.

**Review by doodlewind — APPROVED** [review]
I refactored the naming a bit, thanks for this feat!

**Pull request merged**